### DISCUSSION TITLE
readme是不是有问题呀

### DISCUSSION THREAD

**Discussion by anhuaxiang** [milvus-io/milvus-insight#101] [Q&A] [answered]
这个readme的端口是不是不对呀？
![QQ20210630-172258](https://user-images.githubusercontent.com/26645743/123936970-450b7f00-d9c8-11eb-9e00-8e090ffd35d4.png)

还有还有 
HOST_URL  这个环境变量不起作用，改了在web页面控制台中看到还是访问 http://127.0.0.1:3000
![aaa](https://user-images.githubusercontent.com/26645743/123937302-9ae02700-d9c8-11eb-8236-406cf6ecdd15.png)

**Comment by shanghaikid** [comment, marked as answer]
1. please update your docker image, we just release a new version 
2. milvus-insight is only support milvus 2.0
3. 127.0.0.1 is your docker host url

let me know if you have any questions, many thanks.

**Reply by shanghaikid** [reply]
we'll update the readme, thanks.
